examples/pong/pong.rs

```diff
@@ -120,22 +123,51 @@ fn initialise_paddles(world: &mut World) {
         .with(GlobalTransform::default())
         .with(right_transform)
         .build();
+
+    // Build the sprite for the paddles.
+    let sprite = Sprite {
+        left: 0.0,
+        right: PADDLE_WIDTH,
+        top: 0.0,
+        bottom: PADDLE_HEIGHT,
+    };
+
+    // Add the sprite to the paddles.
+    // This is done in a separate step here because as they are identical sprites,
+    // adding them both at once allows for better performance. Check out the ball
+    // sprite to see how to add a sprite to only one entity.
+    world
+        .exec(|mut data: SpriteRenderData| {
+            data.add_multiple(
+                vec![paddle_left, paddle_right],
+                &sprite,
+                spritesheet,
+                SPRITESHEET_SIZE,
+            )
+        })
+        .expect("Error creating SpriteRender for paddles");
 }
 
 /// Initialises one ball in the middle-ish of the arena.
-fn initialise_balls(world: &mut World) {
-    use {ARENA_HEIGHT, ARENA_WIDTH, BALL_COLOUR, BALL_RADIUS, BALL_VELOCITY_X, BALL_VELOCITY_Y};
+fn initialise_balls(world: &mut World, spritesheet: TextureHandle) {
+    use {ARENA_HEIGHT, ARENA_WIDTH, BALL_RADIUS, BALL_VELOCITY_X, BALL_VELOCITY_Y, PADDLE_WIDTH};
 
-    // Create the mesh, material and translation.
-    let mesh = create_mesh(world, generate_circle_vertices(BALL_RADIUS, 16));
-    let material = create_colour_material(world, BALL_COLOUR);
+    // Create the translation.
     let mut local_transform = Transform::default();
     local_transform.translation = Vector3::new(ARENA_WIDTH / 2.0, ARENA_HEIGHT / 2.0, 0.0);
 
+    // Create the sprite for the ball.
+    let sprite = Sprite {
```

Comment:
I'm really confused here.

Reply:
From my updated pong tutorial:
> We now need to define what part of the spritesheet we want to render.
> To do that, we need to create a `Sprite`, which is a fancy name to call a rectangle on
> the sprite sheet, before the creation of the entities.

Reply:
I was talking about the top left right down coordinates. Sorry I didn't ping the correct line.

Reply:
Ah yes, just give a look at a sprite sheet and it'll be much clearer.
Maybe I could have it be simpler. Any idea?

Reply:
I get it now. Personally I would not use a spritesheet for different / unrelated sprite. If you want to keep it like that its fine, but add a comment explaining how the spritesheet is layouted.